### PR TITLE
Phase 6: srf4L Ensemble Seeds 58-65 (3rd Batch)

### PR DESCRIPTION
**Pull request by tcapelle**
More srf4L seeds to grow the ensemble pool. Seeds 58-65 with --surface_refine_layers 4.

```bash
for seed in 58 59 60 61 62 63 64 65; do
  python train.py --agent nezuko --wandb_group phase6/srf4L-ensemble-v3 \
    --asinh_pressure --asinh_scale 0.75 --surface_refine_layers 4 \
    + all baseline flags --seed ${seed}
done
```

### CODE REVIEW

**Comment by tcapelle** [comment]
ADVISOR: Closing PR #2085 because ADVISOR: Closing — srf4L hypothesis invalidated across two independent experiments. 4-layer surface refine consistently regresses p_tan by +5-7% (askeladd #2079, fern #2083). Your runs will finish but are expected to show the same pattern. Reassigning you to a new experiment..

**Comment by tcapelle** [comment]
STUDENT nezuko:

## Results

### srf4L Ensemble Seeds 58-65 (3rd Batch)

All 8 seeds completed successfully. Each ran for ~180 minutes (wall-clock timeout) reaching ~155-165 epochs with `--surface_refine_layers 4`.

| Seed | p_in | p_oodc | p_tan | p_re | val_loss | W&B Run ID |
|------|------|--------|-------|------|----------|------------|
| 58 | 12.9 | 8.2 | 33.7 | 6.4 | 0.2684 | m4jr73c1 |
| 59 | 13.0 | 8.3 | 32.4 | 6.3 | 0.2613 | i8btzlg5 |
| 60 | 13.6 | 7.7 | 31.0 | 6.6 | 0.2663 | ffewbtsr |
| 61 | 13.1 | 7.7 | 32.8 | 6.3 | 0.2745 | rqef6wcl |
| 62 | 12.9 | 8.7 | 31.3 | 6.6 | 0.2630 | hnh3h2yc |
| 63 | 12.7 | 7.7 | 31.0 | 6.6 | 0.2642 | yffnutnt |
| 64 | 13.3 | 7.9 | 33.4 | 6.5 | 0.2732 | lwzozi4y |
| 65 | 12.7 | 8.1 | 31.9 | 6.4 | 0.2570 | uff5tlry |
| **Mean** | **13.0** | **8.0** | **32.2** | **6.5** | **0.2660** | — |
| **Std** | **0.3** | **0.4** | **1.0** | **0.1** | **0.005** | — |
| **Best** | **12.7** | **7.7** | **31.0** | **6.3** | **0.2570** | — |

### Comparison vs Asinh Baseline (3L, 8-seed mean)

| Metric | srf4L (this batch) | srf3L baseline | Delta |
|--------|-------------------|----------------|-------|
| p_in | 13.0 ± 0.3 | 13.03 ± 0.39 | ~0% |
| p_oodc | 8.0 ± 0.4 | 7.83 ± 0.19 | +2.2% |
| p_tan | 32.2 ± 1.0 | 30.29 ± 0.47 | +6.3% |
| p_re | 6.5 ± 0.1 | 6.45 ± 0.05 | +0.8% |

### Command used
```bash
for seed in 58 59 60 61 62 63 64 65; do
  CUDA_VISIBLE_DEVICES=$((seed - 58)) python train.py --agent nezuko \
    --wandb_name "nezuko/srf4L-s${seed}" --wandb_group "phase6/srf4L-ensemble-v3" \
    --asinh_pressure --asinh_scale 0.75 \
    --field_decoder --adaln_output --use_lion --lr 2e-4 \
    --aug aoa_perturb --aug_full_dsdf_rot --high_p_clamp \
    --n_layers 3 --slice_num 96 --tandem_ramp \
    --domain_layernorm --domain_velhead --ema_decay 0.999 \
    --weight_decay 5e-5 --cosine_T_max 160 --disable_pcgrad \
    --pressure_first --pressure_deep --residual_prediction \
    --surface_refine --surface_refine_hidden 192 --surface_refine_layers 4 \
    --seed ${seed}
done
```

- **W&B group:** `phase6/srf4L-ensemble-v3`
- **Peak VRAM:** ~38 GB per seed
- **Training time:** ~180 min each (hit wall-clock limit)

### What happened

The srf4L seeds show comparable p_in and p_re to the srf3L baseline, but slightly worse p_oodc (+2.2%) and notably worse p_tan (+6.3%). The extra surface refinement layer doesn't provide a clear individual-model benefit over 3 layers — the 4th layer may be overfitting the surface predictions on the tandem transfer split.

However, these seeds are intended for ensemble pooling rather than individual comparison. The diversity from different architectures (3L vs 4L) may provide complementary predictions that improve ensemble performance even if individual models aren't strictly better.

### Suggested follow-ups

- Evaluate mixed-architecture ensembles combining the best srf3L and srf4L seeds to see if diversity improves ensemble accuracy
- The tandem transfer degradation with 4L suggests possible overfitting — could try adding dropout or reducing hidden size for deeper surface refinement
